jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -537,6 +537,13 @@ public Module addEnableNativeAccess(Module mod) {
 		return mod.implAddEnableNativeAccess();
 	}
 
+/*[IF (JAVA_SPEC_VERSION >= 21) & (JAVA_SPEC_VERSION < 24)]*/
+	@Override
+	public boolean allowSecurityManager() {
+		return System.allowSecurityManager();
```

Comment:
https://github.com/eclipse-openj9/openj9/blob/f8939d13ed0f345dd7edb466b927f17bf19b6b01/jcl/src/java.base/share/classes/java/lang/System.java#L1304-L1308

`System. allowSecurityManager()` is needed for JDK21.

Reply:
Right... updated.